hummingbot/connector/exchange/kucoin/kucoin_exchange.pyx

```diff
@@ -694,30 +693,32 @@ cdef class KucoinExchange(ExchangeBase):
                 raise ValueError(f"Buy order amount {decimal_amount} is lower than the minimum order size "
                                  f"{trading_rule.min_order_size}.")
         try:
-            exchange_order_id = await self.place_order(order_id, trading_pair, decimal_amount, True, order_type,
-                                                       decimal_price)
             self.c_start_tracking_order(
                 client_order_id=order_id,
-                exchange_order_id=exchange_order_id,
+                exchange_order_id="",
```

Comment:
By default when the exchange_order_id is still not set it should be None (this can be checked in the code from InFlightOrderBase)

Reply:
ditto